grammars/javascript.cson

```diff
@@ -76,18 +76,19 @@
         'name': 'storage.type.function.js'
       '7':
         'name': 'storage.type.function.js'
-      '8':
-        'name': 'punctuation.definition.parameters.begin.js'
     'comment': 'match stuff like: Sound.prototype.play = function() { … }'
-    'end': '(\\))'
+    'end': '\\}'
     'endCaptures':
-      '1':
-        'name': 'punctuation.definition.parameters.end.js'
+      '0':
+        'name': 'punctuation.definition.function.body.end.js'
     'name': 'meta.function.prototype.js'
```

Comment:
I think this needs to be updated, because the entire function block shouldn't be tokenized as a prototype.